srcpkgs/osmin/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'osmin'
+pkgname=osmin
+version=1.8.7
+revision=1
+_libosmscout_version=1.9.2
+create_wrksrc=yes
+build_style=cmake
+configure_args="-DCMAKE_C_COMPILER=/usr/bin/clang -DCMAKE_CXX_COMPILER=/usr/bin/clang++ ."
+hostmakedepends="qt5-qmake pkg-config qt5-host-tools qt5-remoteobjects git"
+makedepends="marisa-devel libxml2-devel qt5-declarative-devel qt5-3d clang libgomp-devel
```

Comment:
clang in makedepends looks a bit suspicious, is it really required here?

Reply:
Without clang the package is a bit bigger but not by much. I'm removing it.

Reply:
When running it, I get a segmentation fault. I'm going back to use clang...